api/external_resources/models.py

```diff
@@ -0,0 +1,18 @@
+from django.db import models
+
+from features.models import Feature
+
+
+class ExternalResources(models.Model):
```

Comment:
What is an external resource? Can this be used for anything other than GitHub?

Reply:
The external resource model is intended to be something generic in case there are other integrations with the app.